packages/webapp/src/components/LocationDetailLayout/Sensor/SensorDetail/index.jsx

```diff
@@ -0,0 +1,230 @@
+            path: `/sensor/${location_id}/details`,
+          },
+        ]}
+      />
+      <Input
+        label={t('SENSOR.DETAIL.NAME')}
+        style={{ paddingBottom: '32px', paddingTop: '24px' }}
+        disabled={true}
+        value={name}
+      />
+      <div
+        className={'latLong'}
+        style={{
+          flexDirection: 'row',
+          display: 'inline-flex',
+          paddingBottom: '32px',
+          width: '100%',
+          gap: '16px',
+        }}
+      >
+        <Input
+          label={t('SENSOR.DETAIL.LATITUDE')}
+          disabled={true}
+          value={point.lat}
+          classes={{ container: { flexGrow: 1 } }}
+        />
+        <Input
+          label={t('SENSOR.DETAIL.LONGITUDE')}
+          disabled={true}
+          value={point.lng}
+          classes={{ container: { flexGrow: 1 } }}
+        />
+      </div>
+      <label>{t('SENSOR.READING.TYPES')}</label>
+      <div className={clsx(styles.container)}>
+        <div className={clsx(styles.pillContainer)}>
+          {sensor_reading_types.map((type) => {
+            return (
+              <Pill
+                key={type}
+                label={t(`SENSOR.READING.${type.toUpperCase()}`)}
+                removable={false}
+                className={'activePill'}
+                selected={true}
+              />
+            );
+          })}
+        </div>
+      </div>
+
+      <Unit
+        register={register}
+        defaultValue={depth}
+        label={t('SENSOR.DETAIL.DEPTH')}
+        hookFormSetValue={setValue}
+        hookFormGetValue={getValues}
+        hookFromWatch={watch}
+        name={t('SENSOR.DETAIL.DEPTH')}
+        unitType={container_planting_depth}
+        max={10000}
+        system={system}
+        control={control}
+        style={{ paddingBottom: '32px' }}
+        disabled={true}
+      />
+
+      <ReactSelect
+        label={t('SENSOR.DETAIL.BRAND')}
+        placeholder={brand_name}
+        defaultValue={brand_name}
+        isDisabled={true}
+        style={{ paddingBottom: '32px' }}
+        toolTipContent={t('SENSOR.DETAIL.BRAND_TOOLTIP')}
+      />
+
+      <Input
+        label={t('SENSOR.DETAIL.MODEL')}
+        style={{ paddingBottom: '32px' }}
+        disabled={true}
+        optional={true}
+        value={model}
+      />
+
+      <Input
+        label={t('SENSOR.DETAIL.EXTERNAL_IDENTIFIER')}
+        style={{ paddingBottom: '32px' }}
+        disabled={true}
+        optional={true}
+        toolTipContent={t('SENSOR.DETAIL.EXTERNAL_ID_TOOLTIP')}
+        value={external_id}
+      />
+      {/* <Input
+        label={t('SENSOR.DETAIL.PART_NUMBER')}
+        style={{ paddingBottom: '32px' }}
+        disabled={true}
+        optional={true}
+        value={PART_NUMBER}
+      />
+      <Input
+        label={t('SENSOR.DETAIL.HARDWARE_VERSION')}
+        style={{ paddingBottom: '32px' }}
+        disabled={true}
+        optional={true}
+        value={HARDWARE_VERSION}
+      /> */}
```

Comment:
Do we want to get rid of this commented out code?

Reply:
@smattingly What do you think? We talked about removing the part number & hardware version from the details tab for now, but if we want to add those back later we could simply comment these lines out.